crates/holochain_sqlite/Cargo.toml

```diff
@@ -12,9 +12,6 @@ edition = "2021"
 [dependencies]
 async-trait = "0.1"
 anyhow = "1.0"
-# pinned here with = to fix a compilation issue in a dependent crate.
-# feel free to drop the = if it causes problems in the future.
-chashmap = "=2.2.0"
```

Comment:
Vague comment, not sure if this is still a problem but it's been here long enough so I'm going to try it and see...

Reply:
yay, killit